ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/424_model_serving_bias_metrics.robot

```diff
@@ -0,0 +1,151 @@
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=${aws_bucket}
+    Create Model Server    token=${FALSE}    server_name=${RUNTIME_NAME}
+
+Bias Metrics Suite Teardown
+    [Documentation]     Bias Metrics Suite Teardown
+    Delete Data Science Project From CLI    displayed_name=${PRJ_TITLE}
+    RHOSi Teardown
+
+Verify User Workload Monitoring Configuration
+    [Documentation]    Verifies that the ALL monitoring components for user-defined-projects
+    ...    are READY in the ${DEFAULT_MONITORING_NS} namespace
+    Wait For Pods Status  namespace=${DEFAULT_MONITORING_NS}  timeout=60
+    Log  Verified Applications NS: ${DEFAULT_MONITORING_NS}  console=yes
+
+Install And Verify TrustyAI Service
+    [Documentation]    Install TrustyAI service CRD and verify that TrustyAI resources have spun up
+    Deploy Custom Resource    kind=TrustyAIService    namespace=${PRJ_TITLE}
+    ...    filepath=${TRUSTYAI_CR_FILEPATH}
+    ${return_code}    ${output}    Run And Return Rc And Output   oc get route trustyai-service -n ${PRJ_TITLE} --template={{.spec.host}}
+    Should Be Equal As Integers    ${return_code}	 0
+    Set Suite Variable    ${TRUSTY_ROUTE}    https://${output}
+
+Remove TrustyAI Service
+    [Documentation]    Remove TrustyAI service CRD
+    OpenshiftLibrary.Oc Delete    kind=TrustyAIService    namespace=${PRJ_TITLE}    name=trustyai-service
+
+Verify Model Is Registered with TrustyAI Service
+    [Documentation]    Verify the deployed model is registered with TrustyAI Service by verifying MM_PAYLOAD_PROCESSORS
+    ...                value is the TrustyAI service route
+    ${MM_PAYLOAD_PROCESSORS_Expected}=    Set Variable    http://trustyai-service.${PRJ_TITLE}.svc.cluster.local/consumer/kserve/v2
+    Wait Until Keyword Succeeds  3 min  20 sec  Verify One Model Serving Pod Exists   ${PRJ_TITLE}    label_selector=modelmesh-service=modelmesh-serving
+    ${podname}=    Get Pod Name   ${PRJ_TITLE}    label_selector=modelmesh-service=modelmesh-serving
+    Log    Serving Runtime Podname: ${podname}
+    ${MM_PAYLOAD_PROCESSORS_Actual} =  Run  oc get pod ${podname} -n ${PRJ_TITLE} -o json | jq '.spec.containers[0].env[] | select(.name=="MM_PAYLOAD_PROCESSORS") | .value'
+    ${MM_PAYLOAD_PROCESSORS_Actual} =    Strip String    ${MM_PAYLOAD_PROCESSORS_Actual}    characters="
+    Should Be Equal  ${MM_PAYLOAD_PROCESSORS_Actual.strip()}  ${MM_PAYLOAD_PROCESSORS_Expected.strip()}
+
+Verify One Model Serving Pod Exists
+    [Documentation]    Verifies that serving runtime pods have been stabilized and only 1 pod exists
+    [Arguments]    ${PRJ_TITLE}     ${label_selector}
+    ${return_code}    ${pod_number}    Run And Return Rc And Output   oc get pod -n ${PRJ_TITLE} -l ${label_selector} | tail -n +2 | wc -l
+    ${pod_numbers}    Split String     ${pod_number}
+    Should Be Equal    ${pod_numbers}[0]   1
```

Comment:
could you move this to a common resource file? maybe making the number of replicas a parameter